### PR TITLE
Use stream sorting.

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Use the Java 8 Stream API's sorting methods.

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by litetex — COMMENTED** [review]
Code LGTM; How can I test these changes?

**Comment by Isira-Seneviratne** [comment]
> Code LGTM; How can I test these changes?

The tags can be viewed in any video under the description tab, like below:



As for the other change, it's related to the playlist functionality.

**Review by litetex — APPROVED** [review]

File: app/src/main/java/org/schabi/newpipe/fragments/detail/DescriptionFragment.java [hunk 1]
```diff
@@ -209,8 +208,8 @@ private void addTagsMetadataItem(final LayoutInflater inflater, final LinearLayo
             final ItemMetadataTagsBinding itemBinding
                     = ItemMetadataTagsBinding.inflate(inflater, layout, false);
 
-            final List<String> tags = new ArrayList<>(streamInfo.getTags());
-            Collections.sort(tags);
+            final List<String> tags = streamInfo.getTags().stream().sorted()
+                    .collect(Collectors.toList());
             for (final String tag : tags) {
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/fragments/detail/DescriptionFragment.java:213]: Wouldn't this work?
```suggestion
            streamInfo.getTags().stream().sorted().forEach(tag -> {
```

**Review by Stypox — APPROVED** [review]

**Comment by Hanankhan1983** [comment]
> > Code LGTM; How can I test these changes?
> 
> The tags can be viewed in any video under the description tab, like below:
> 
> 
> 
> As for the other change, it's related to the playlist functionality.
> 
> 

Pls download link :(

**Comment by Hanankhan1983** [comment]
> Code LGTM; How can I test these changes?

Pls download link

**Comment by Stypox** [comment]
@Hanankhan1983 I don't get why you would want a download link for this pull request. Nothing has changed regarding users, this is just an internal change.

Anyway, follow the steps in the "APK testing" section above (in the PR description) to obtain an apk.

**Comment by Hanankhan1983** [comment]
> @Hanankhan1983 I don't get why you would want a download link for this pull request. Nothing has changed regarding users, this is just an internal change.
> 
> Anyway, follow the steps in the "APK testing" section above (in the PR description) to obtain an apk.

You are showing in the picture? That we can see any video tags and ect?

**Comment by Stypox** [comment]
@Hanankhan1983 you can already view video tags in the current version (and even in older ones), just scroll down the description tab.

Those screenshots were sent just to show which features needed to be tested to make sure the under-the-hood changes this Pull Request made actually worked correctly.

**Comment by Hanankhan1983** [comment]
> @Hanankhan1983 you can already view video tags in the current version (and even in older ones), just scroll down the description tab.
> 
> Those screenshots were sent just to show which features needed to be tested to make sure the under-the-hood changes this Pull Request made actually worked correctly.

Ok

**Comment by opusforlife2** [comment]
@Hanankhan1983 Don't quote entire text blocks unless you're referring to something in them for context.

**Pull request merged**